packages/components/src/components/page/page.scss

```
@@ -18,15 +18,15 @@
 		display: flex;
 		flex-direction: column;
 
-		header,
-		footer {
+		header:first-of-type,
+		footer:first-of-type {
```

```suggestion
		> :is(header, footer) {
```

I assume that you mean by this the first header or footer within the document or directly underneath the page. But any nested `header` or `footer` would also be targeted by this selector, as it mainly means "on the same level", compare to e.g. https://jsbin.com/xetuxovoru/edit?html,css,output

---

fixed

---

There is no header showcase for angular I need to test it as well...